l1-contracts/contracts/state-transition/chain-deps/facets/Executor.sol

```diff
@@ -457,20 +449,10 @@ contract ExecutorFacet is ZkSyncHyperchainBase, IExecutor {
     /// @dev Gets zk proof public input
     function _getBatchProofPublicInput(
         bytes32 _prevBatchCommitment,
-        bytes32 _currentBatchCommitment,
-        VerifierParams memory _verifierParams
+        bytes32 _currentBatchCommitment
     ) internal pure returns (uint256) {
         return
-            uint256(
-                keccak256(
-                    abi.encodePacked(
-                        _prevBatchCommitment,
-                        _currentBatchCommitment,
-                        _verifierParams.recursionNodeLevelVkHash,
-                        _verifierParams.recursionLeafLevelVkHash
-                    )
-                )
-            ) >> PUBLIC_INPUT_SHIFT;
+            uint256(keccak256(abi.encodePacked(_prevBatchCommitment, _currentBatchCommitment))) >> PUBLIC_INPUT_SHIFT;
```

Comment:
if we are taking the verifier params from the verifier contract do we need to add them in here before hashing/shifting?

Reply:
it is just that the public input used to contain those verifier params too